taca/__init__.py

```diff
@@ -1,4 +1,4 @@
 """ Main TACA module
 """
 
-__version__ = '0.5.0'
+__version__ = '0.5.1'
```

Comment:
@kate-v-stepanova : remember to (pull and) change the version number to right one when you make final push 👍